python/dbsp/program.py

```diff
@@ -17,9 +17,10 @@ class DBSPProgram:
     compiler.
     """
 
-    def __init__(self, api_client, program_id, program_version):
+    def __init__(self, api_client, program_id, program_name, program_version):
```

Comment:
Can we get rid of program_id here?

Reply:
Maybe longer term, as the Python will be refactored soon I think?

Reply:
Yes, my goal with the python bindings was to just get it to pass tests for now. We're going to remove all of it anyway.